### PR TITLE
Retired Emails SDK Changes

### PR DESCRIPTION
**Pull request by jcook-stytch**

### CODE REVIEW

File: stytch/b2b/organizations/types.go [hunk 1]
```diff
@@ -484,7 +485,7 @@ type SCIMRegistration struct {
 	// ExternalID: The ID of the member given by the identity provider.
 	ExternalID string `json:"external_id,omitempty"`
 	// SCIMAttributes: An object for storing SCIM attributes brought over from the identity provider.
-	SCIMAttributes map[string]any `json:"scim_attributes,omitempty"`
+	SCIMAttributes *scim.SCIMAttributes `json:"scim_attributes,omitempty"`
```
Comment (jennifer-stytch) [inline stytch/b2b/organizations/types.go:488]: This is technically a breaking change, so we might need to make this a major version bump

**Review by jennifer-stytch — APPROVED** [review]

**Comment by jennifer-stytch** [comment]
@ollie-stytch @bgier-stytch are we good to release all the SCIM stuff?

File: stytch/b2b/organizations/types.go (hunk 1 above)
Comment (ollie-stytch) [reply stytch/b2b/organizations/types.go:488]: This is broken in it's current state. No one is using the scim client libraries so I think we're ok to minor bump as we're essentially just adding functionality ?

Comment (logan-stytch) [reply stytch/b2b/organizations/types.go:488]: Has SCIM been released? If so, I would say this is a MAJOR bump. But if it's truly broken (and has never worked), I'm okay calling it MINOR.

Comment (jcook-stytch) [reply stytch/b2b/organizations/types.go:488]: Talked to @ollie-stytch today and I agree with him and @logan-stytch that it makes the most sense for this to be a minor bump - there's a breaking change in types, but the endpoint has never worked correctly and has no users.

@jennifer-stytch if that sounds good to you should we 🚢 ?

Comment (jennifer-stytch) [reply stytch/b2b/organizations/types.go:488]: sgtm! :shipit:

**Review by jennifer-stytch — APPROVED** [review]

**Pull request merged**